app/lib/lnd/config.js

```diff
@@ -293,6 +310,31 @@ class LndConfig {
       debug('Connection type is local. Assigning settings as: %o', defaultLocalOptions)
       Object.assign(this, defaultLocalOptions)
     }
+
+    // Set the ready prop once we have calculated all of the config properties.
+    const ready = new Promise(async resolve => {
```

Comment:
Consider avoiding using `new Promise` here. I believe this can be done in a simpler and more readable way like this: (haven't tested this code)

```
const isReady = async () => {
    try {
        if (this.type === 'local') {
            // We don't support lndconnect for local nodes.
            return Promise.resolve(true)
        }

        const { host, cert, macaroon } = this
        const [encodeCert, encodeMacaroon] = await Promise.all([
          lndconnect.encodeCert(cert),
          lndconnect.encodeMacaroon(macaroon)
        ])
        const connectionstring = lndconnect.encode({
          host,
          cert: encodeCert,
          macaroon: encodeMacaroon
        })
        _lndConnect.set(this, connectionstring)
        return Promise.resolve(true)
    }
    catch {
        return Promise.resolve(true)
    }
}

_ready.set(this, isReady())
```

Reply:
Have updated this to use `async`